resources/js/app.js

```diff
@@ -12,16 +32,17 @@
 		})
 	})
 
-	app.controller('TabController', function () {
-		this.tab = 1
-		this.setTab = function (stab) {
-			this.tab = stab
-			$(".button-collapse").sideNav('hide');
-		}
-		this.isSet = function (stab) {
-			return this.tab == stab
-		}
-	})
+    app.controller('TabController', function ($location) {
+        this.tab = "/home";
```

Comment:
We are giving a default path here, thats why every time the page is loaded and hence this controller is loaded,  this.tab gets set to '/home' no matter what the path in url is. So we need to replace this with 
```
 this.tab = $location.path()
```